### PR TITLE
Reviewer 2 edits

### PR DESCRIPTION
**Pull request by klwetstone**
Incorporate [edits from reviewer 2](https://github.com/scipy-conference/scipy_proceedings/pull/936#pullrequestreview-2201094391)

One outstanding question: How many of the training data points had valid satellite imagery? Where would we have this documented?
- In the performance section, we note that 2,880 of the 4,035 test set observations could be used because of imagery availability.
- I added a placeholder sentence for how I think we should express this for the training set, but couldn't find the exact number.

### CODE REVIEW

**Comment by ejm714** [comment]
> How many of the training data points had valid satellite imagery? Where would we have this documented?

@klwetstone We see this from the number of sample IDs in the sentinel training metadata that's saved to s3 (thanks past selves!). It's 5,721, I'll add it in.

**Review by ejm714 — APPROVED** [review]
Thanks so much for the draft edits @klwetstone! This made polishing it off so much easier
